accounting/docs.md

```diff
@@ -102,3 +102,33 @@ Circular receipts record instances where we have paid one of our own invoices.
 - TxID: The payment hash of the invoice.
 - Reference: The preimage of the invoice.
 - Note: Optionally set if the invoice had a memo attached, was overpaid, or was a keysend.
+
+### Payment
+Payments off chain represent payments made via the Lightning Network. 
+
+- Amount: The amount that we paid, excluding the off chain fees paid. 
```

Comment:
I know, this is pre-existing for the other parts in this document, but it would probably be good to mention this is the amount in millisats, even for on-chain entries.